Cargo.toml

```diff
@@ -1,20 +1,16 @@
 [package]
-name = "reflink"
-version = "0.1.3"
-authors = ["Nicolas Koch <nioko1337@gmail.com>"]
+name = "reflink-copy"
+version = "0.1.4"
+authors = ["Jiahao XU <Jiahao_XU@outlook.com>"]
 edition = "2018"
 description = "copy-on-write mechanism on supported file systems"
-documentation = "https://docs.rs/reflink/*/reflink/"
-homepage = "https://github.com/nicokoch/reflink"
-repository = "https://github.com/nicokoch/reflink"
+documentation = "https://docs.rs/reflink-copy"
+homepage = "https://github.com/cargo-bins/reflink"
+repository = "https://github.com/cargo-bins/reflink"
```

Comment:
those are the old links (i've renamed this repo!)

Reply:
I've updated the links!